fixtures/apps_fixtures.json

```diff
@@ -54,6 +54,24 @@
     "model": "apps.apps",
     "pk": 10
   },
+  {
+    "fields": {
+      "category": "develop",
```

Comment:
I'm not sure, that this is the right category for the app, it seems like it will be a public app down the line, so it'd make sense for it to be `serve` category.
I'd also move to the bottom as the next `serve` type application.

But we can think about it:)